package/yast2.changes

```diff
@@ -1,4 +1,29 @@
 -------------------------------------------------------------------
+Thu Oct 26 09:03:03 UTC 2017 - knut.anderssen@suse.com
+
+- Network (Firewall):
+  - Added support for firewalld offline command
+  - Dropped SuSEFirewall (module) support for SuSEFirewall2 backend
```

Comment:
As a user point of view, I do not see clearly if SuSEFirewall2 backend is dropped or not.

Reply:
SuSEFirewall2 class is still available but:

There is no support for multiple bakends in SuSEFirewall module anymore:
https://github.com/yast/yast-yast2/pull/635/files#diff-fbf84b02e6a280d30bd581f1cd6c477dR34

Previously it checked the running firewall using SuSEFirewall2 as fallback by default, now SuSEFirewall module will always use SuSEFirewalld.

But that is just temporary, we will completely remove all these classes/modules or at least that is the idea.